### DISCUSSION TITLE
Subdomains and payload.hostname must match the following: "/^(([a-zA-Z0-9]| ..."

### DISCUSSION THREAD

**Discussion by joshy** [umami-software/umami#2728] [Q&A]
Hi,
I have maybe a stupid question. I have several small application running on _*.imaging.ms.uhbs.ch_. However I can't add a specific website via settings. For example adding: _foo.imaging.ms.uhbs.ch_ gives me the error: `Invalid domain. Do not include http/https.`

I ignored the message and added the following site to umami: _imaging.ms.uhbs.ch_. Maybe I can later differentiate.

Running the tracking on _foo.imaging.ms.uhbs.ch_ gives the following error when sending a HTTP Post to _https://analytics.ict-mscc.uhbs.ch/api/send_:
`payload.hostname must match the following: "/^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9-]*[A-Za-z0-9])$/"` with a HTTP 400. 

Thanks for the help and with best regards,
Joshy

**Comment by QYG2297248353** [comment]
hostname only supports ipv4 and domain names, and does not contain protocols

If it is an ipv6 host address, I'm sorry, it's not currently supported
#2862

**Reply by joshy** [reply]
It is not an ipv6 address as far as I can tell. My assumptions is that it doesn't support subdomains.

**Reply by QYG2297248353** [reply]
I think you should test this regex
![image](https://github.com/user-attachments/assets/1594649c-8974-46a3-9e0d-0701911ce2d8)

**Comment by joshy** [comment]
The regex doesn't come from me but umami. :)
